src/transformers/models/luke/tokenization_luke.py

```diff
@@ -0,0 +1,1412 @@
+                The sequence or batch of sequences to be encoded. Each sequence must be a string. Note that this
+                tokenizer does not support tokenization based on pretokenized strings.
+            entity_spans (:obj:`List[Tuple]`, :obj:`List[List[Tuple]]`, `optional`):
+                The sequence or batch of sequences of entity spans to be encoded. Each sequence consists of tuples each
+                with two integers denoting start and end positions of entities. If you specify "entity_classification"
+                or "entity_pair_classification" as the `task` argument in the constructor, the length of each sequence
+                must be 1 or 2, respectively. If you specify `entities`, the length of each sequence must be equal to
+                the length of each sequence of `entities`.
+            entity_spans_pair (:obj:`List[Tuple]`, :obj:`List[List[Tuple]]`, `optional`):
+                The sequence or batch of sequences of entity spans to be encoded. Each sequence consists of tuples each
+                with two integers denoting start and end positions of entities. If you specify the `task` argument in
+                the constructor, this argument is ignored. If you specify `entities_pair`, the length of each sequence
+                must be equal to the length of each sequence of `entities_pair`.
+            entities (:obj:`List[str]`, :obj:`List[List[str]]`, `optional`):
+                The sequence or batch of sequences of entities to be encoded. Each sequence consists of strings
+                representing entities, i.e., special entities (e.g., [MASK]) or entity titles of Wikipedia (e.g., New
+                York). This argument is ignored if you specify the `task` argument in the constructor. The length of
+                each sequence must be equal to the length of each sequence of `entity_spans`. If you specify
+                `entity_spans` without specifying this argument, the entity sequence or the batch of entity sequences
+                is automatically constructed by filling it with the [MASK] special entities.
+            entities_pair (:obj:`List[str]`, :obj:`List[List[str]]`, `optional`):
+                The sequence or batch of sequences of entities to be encoded. Each sequence consists of strings
+                representing entities, i.e., special entities (e.g., [MASK]) or entity titles of Wikipedia (e.g., New
+                York). This argument is ignored if you specify the `task` argument in the constructor. The length of
+                each sequence must be equal to the length of each sequence of `entity_spans_pair`. If you specify
+                `entity_spans_pair` without specifying this argument, the entity sequence or the batch of entity
+                sequences is automatically constructed by filling it with the [MASK] special entities.
+            max_entity_length (:obj:`int`, `optional`):
+                The maximum length of :obj:`entity_ids`.
```

Comment:
Please remember that objects between \` \` will not render in code-formatting but in italics. You need to use :obj:\`xxx\` or \`\`xxx\`\`